configuration/redis/redis_test.go

```diff
@@ -118,9 +120,43 @@ func TestConfigurationStore_Get(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "test does not throw error for wrong type during get all",
+			prepare: func(client *redis.Client) {
+				client.HSet(context.Background(), "notSupportedType", []string{"key1", "value1", "key2", "value2"})
```

Comment:
What does this line do?

Reply:
This line writes an unsupported value to redis. After that, the test case try to get all values. and skip returning this unsupported value.

Reply:
Got it. Since string slice is an unsupported type, should we change it to this to avoid confusion with keys and values?
```suggestion
				client.HSet(context.Background(), "testKeyUnsupported", []string{"value1", "value2", "value3"})
```

Reply:
https://redis.io/docs/manual/data-types/ although it looks like lists are supported by Redis.